### PR TITLE
[PLAT-937] Adds methods for secure compute endpoints

### PR DESCRIPTION
**Pull request by lily-tsou**
- Adds methods for secure compute endpoints
- Adds tests for new methods

### CODE REVIEW

File: types/computation.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Computation {
```
Comment (lily-tsou) [inline types/computation.js:1]: Forgot to write a comment here. Will do that before merging!

File: types/computation.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+    static decode(json){
+        if("computations" in json)
+            return new Computation(json.computations)
+        // If json is an empty object, create a new Computation object with an empty array
+        return new Computation([])
+    }
+}
+
+  module.exports = Computation
```
Comment (lily-tsou) [inline types/computation.js:14]: And will add newline :)

**Comment by Mrhea** [comment]
And my sdk methods are coming soon!

File: lib/storage/client.js [hunk 3]
```diff
@@ -1543,5 +1544,81 @@ class Client extends CryptoConsumer {
...
+            throw new Error('Unauthorized')
+        default:
+          throw new Error('Error fetching computations.')
+      }
+    }
+
+    /**
+     * fetchAvailableCOmputations fetches all available computations to the TozStore Client
+     */
+    async fetchAvailableComputations() {
```
Comment (rosasck) [inline lib/storage/client.js:1576]: feel free to push this to master to unblock matthew but can you add documentation to the read me!

**Review by Mrhea — APPROVED** [review]

**Pull request merged**